spec/views/groups/new.html.erb_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature 'Group/Category new page', type: :feature do
+  let!(:user) { create(:user) }
+  let!(:category) { create(:group, name: 'Apple Test', icon: 'https://avatars.githubusercontent.com/u/53082877?s=96&v=4', user:) }
+  before do
+    sign_in user
+    visit new_group_path
+  end
+
+  scenario 'Displays new category page' do
+    expect(page).to have_current_path(new_group_path)
+    expect(page).to have_content('Create New Category')
+  end
```

Comment:
- [ ] You are having a failing test here. Kindly work on it as it's the only test falling in all the tests written 👍🏽

<img width="690" alt="Screenshot 2023-03-31 at 10 26 41 AM" src="https://user-images.githubusercontent.com/50941074/229098561-7a5579a4-87bf-4c9a-abcd-68c6c2dac35d.png">